packages/shared/src/polyfills/styleShim/index.native.ts

```
@@ -0,0 +1 @@
+// nothing;
```

ESLint

[error] Insert `⏎`

(prettier/prettier)